docs/book/user-guide/starter-guide/cache-previous-executions.md

```diff
@@ -166,7 +166,7 @@ if __name__ == "__main__":
     # This will disable cache for all steps
```

Comment:
Since you spotted the mistake here, I took another look at the entire example and I already see a few more lines we can fix here:

- The comment on this line is incorrect. The following code only disables the cache for the `svc_trainer`.
- Additionally, there is another comment a few lines above this one which is also misleading. **There is no `enable_cache=False` in the `@step` decorator.** The statement `ZenML will detect a different value for the
gamma input of the second step and disable caching for that step.` is correct though.